__tests__/jsonConverter-test.js

```diff
@@ -0,0 +1,39 @@
+/*
+  Run via jest jsonConverter-test.js
```

Comment:
We should probably add some tests that test the dialog boxes, but I can do that once I do the select multiple export option feature